dydx/client.py

```diff
@@ -1,17 +1,18 @@
+import json
 import random
 import requests
 import dydx.util as utils
-
 from web3 import Web3
+from .exceptions import DydxAPIError
 
 
 class Client(object):
 
-    TAKER_ACCOUNT_OWNER = '0x0000000000000000000000000000000000000000'
-    TAKER_ACCOUNT_NUMBER = '0'
-    BASE_API_URI = 'https://api.dydx.exchange/v1/dex/'
+    TAKER_ACCOUNT_OWNER = '0xf809e07870dca762B9536d61A4fBEF1a17178092'
+    TAKER_ACCOUNT_NUMBER = 0
+    BASE_API_URI = 'https://api.dydx.exchange/v1/'
 
-    def __init__(self, private_key, public_address=None, account_number='0'):
+    def __init__(self, private_key, public_address=None, account_number=0):
```

Comment:
Should we worry about `account_number` overflowing or just assume most people will use 0? Our isolated accounts on the app can be really big.

Reply:
python will convert to long (unlimited size) in 2.7 when it gets too large.
python3 has unlimited size ints